out_writeapi.go

```diff
@@ -239,6 +239,7 @@ func buildStream(ctx context.Context, config **outputConfig) error {
 		managedwriter.EnableWriteRetries((*config).enableRetry),
 		managedwriter.WithMaxInflightBytes((*config).maxQueueBytes),
 		managedwriter.WithMaxInflightRequests((*config).maxQueueRequests),
+		managedwriter.WithTraceID("FluentBitConnector"),
```

Comment:
Let's keep it short, just `FluentBit`. It will consume space in our logging DB, but also should be readable.

Reply:
FBPlugin?

Reply:
FB is Facebook :)
`FluentBit` is good. The words `connector` and `plugin` don't add new information because we already know it refers to the origin of the request. That's why the name of the connector is enough.

Reply:
Okay sounds good.

Reply:
Done.